custom-graders/README.md

```diff
@@ -43,39 +69,37 @@ Sending build context to Docker daemon
+
+Here is a simple command to test a grader locally:
 ```sh
-{"isCorrect": true,"feedback": "Congrats! All test cases passed!"}
+$ export CustomGraderPath=$BASE_PATH/programming-assignments-demo/custom-graders/
+$ courseraprogramming grade local factoring_grader.v1.1 $CustomGraderPath/DemoAssignmentGrader/SampleSubmission/FactoringSampleSubmission
 ```
 
-### Test graders locally using 'courseraprogramming'
-Testing in Coursera's production environment is quite different from testing the graders locally. 'courseraprogramming' is a software development toolkit that helps to develop asynchronous graders for Coursera (typically programming assignments). It can be used to test the graders locally in an environment that is somewhat similar to Coursera's production environment. Follow instructions [here](https://github.com/coursera/courseraprogramming) to get started.
-
-### Package the docker image into a tar.
+**After your docker grader passes all tests in *courseraprogramming*, its time to upload your docker grader via our authoring tools.**
+#### 1. Package the docker image into a tar
 ```sh
-$ docker save factoring_grader.v1.1 > factoring_grader.v1.1.tar
+$ docker save demo_grader.v1.1 > demo_grader.v1.1.tar
 ```
 
-### Test your grader in Coursera's production environment.
-###### - Upload the docker grader image via Coursera's authoring tools
-###### - Review and Publish the new programming assignment
-###### - Click on 'View as Learner' to interact with your Course as a learner
-###### - Navigate to the programming assignment and make test submissions
-###### - For debugging, copy runId exposed in the submission history
-*This information is just exposed to the course instructor*
-###### Use these runIDs as an input to this [tool](http://52.2.120.167/) to read stdout and stderr output by the docker grader
-*Contact programming@coursera.org to get access credentials for the tool.*
-
-#### Common bugs/issues:
-- One of the most common issues with docker files not working on Coursera's platform is due to setting up inappropriate permissions in the Dockerfile. Coursera's infrastructure executes docker images as non-root users without any network access for security reasons and its important to set permissions carefully for files/directories that will be read/written/executed inside the docker container.
-- JSON output doesn't exactly match the specified format which causes Grader failures. The docker grader should only write the JSON object specified above to stdout and nothing else. 
-- Use of environment variables as part of the grader is not prohibited (i.e. you can set them in your grader files) but we do limit the ability to set environment variables as part of the 'Dockerfile' for security purposes. Hence at runtime, the grader code will not be able to access any environment variables that have been set directly in the 'Dockerfile'.
+#### 2. Upload the grader to production course
```

Comment:
I think we should migrate this over to use `courseraprogramming` eventually, but not required for this PR.